src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Clear cache pools.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolClearCommand extends ContainerAwareCommand
```

Comment:
> Github provides a saved reply feature

please don't do that :)

>  If you don't add final, you agree with BC for those who wants to extend this command?

YES! That's written, and reviewers have this very important policy in mind all the time.
See http://symfony.com/bc